pkg/asset/installconfig/powervs/client.go

```diff
@@ -1178,3 +1182,113 @@ func (c *Client) CreateSSHKey(ctx context.Context, serviceInstance string, zone
 
 	return nil
 }
+
+// AddIPToLoadBalancerPool adds a server to a load balancer pool for the specified port.
+func (c *Client) AddIPToLoadBalancerPool(ctx context.Context, lbID string, poolName string, port int64, ip string) error {
```

Comment:
From your description, this will eventually be implemented upstream. If there is an issue, it would be good to add a reference to it here with a TODO comment:
```suggestion
// TODO: remove once <ibmcloud-issue-link> is implemented
// AddIPToLoadBalancerPool adds a server to a load balancer pool for the specified port.
func (c *Client) AddIPToLoadBalancerPool(ctx context.Context, lbID string, poolName string, port int64, ip string) error {
```
If there isn't, then it should be created :)